uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Validator/Date.php

```diff
@@ -21,13 +19,10 @@ class CRM_HRLeaveAndAbsences_Validator_Date {
    *
    * @return bool
    */
-  public static function isValid($date, $format = 'Y-m-d')
+  public static function isValid($date, $format = 'YmdHis')
```

Comment:
Do the tests in CRM_HRLeaveAndAbsences_BAO_DateTest still pass after this change?